org.eclipse.winery.topologymodeler.ui/package-lock.json

```diff
@@ -7,7 +7,7 @@
     "@angular-devkit/build-optimizer": {
       "version": "0.0.42",
       "resolved": "https://registry.npmjs.org/@angular-devkit/build-optimizer/-/build-optimizer-0.0.42.tgz",
-      "integrity": "sha512-BAYCVZ10ro6mgZQDZiNiVbX8ppygw4q7z/stpwG8WjMswgMRIcxsxYoC1VFuWcUPAf4UyfTIav6e8UZWA5+xnQ==",
```

Comment:
Please ignore these changes. We require the sha512 hash.

Reply:
done